core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java

```diff
@@ -264,6 +270,16 @@ private TableUpdateDetails getTableUpdateDetailsFromSharedArea(@NotNull NetworkI
             } else {
                 int status = engine.getStatus(securityContext, path, tableNameUtf16, 0, tableNameUtf16.length());
                 if (status != TableUtils.TABLE_EXISTS) {
+                    if (!autoCreateNewTables) {
```

Comment:
can we please reuse the code? same code and error messages are in `initCacheRecord()`

Reply:
`initCacheEntry` is in Line UDP family. There is quite some duplications between line TCP and UDP. How would you suggest to reuse exception building code between them?